test/suite/test_sweep03.py

```diff
@@ -127,14 +126,11 @@ def test_disable_idle_timeout_drop_force(self):
 
         # Grab the stats post table drop to see things have decremented
         stat_cursor = self.session.open_cursor('statistics:', None, None)
-        cache2 = stat_cursor[stat.conn.cache_bytes_inuse][2]
         close2 = stat_cursor[stat.conn.dh_sweep_close][2]
         stat_cursor.close()
 
-        # Ensure that the handle has been closed after the drop.
-        self.assertEqual(close2, 1)
-        # Ensure that any space was reclaimed from cache.
-        self.assertLess(cache2, cache1)
+        # Handles aren't closed immediately on a force drop, neither is memory immediately freed.
+        self.assertEqual(close2, close1)
```

Comment:
I don't understand this.  I was under the impression that the drop itself closed the btree, and sweep no longer tries to close the dropped handle.  If the sweep isn't closing it, how and when does it get closed?  Is there any way to test it to make sure it does get closed?

Reply:
I agree that looks a bit odd, and I'll take a deeper look.